chat-client/src/client/messager.ts

```diff
@@ -44,7 +45,9 @@ export interface OutboundChatApi {
 export class Messager {
     constructor(private readonly chatApi: OutboundChatApi) {}
 
-    onTabAdd = (tabId: string): void => {
+    onTabAdd = (tabId: string, triggerType?: TriggerType): void => {
+        this.chatApi.telemetry({ triggerType: triggerType ?? 'click', tabId, name: 'tabAdd' })
```

Comment:
Have you considered defining this telemetry on server side based on `tabAdded` event?